language-design.md

```diff
@@ -38,6 +96,43 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+There are many ways to define a well-designed language. We believe that 
+language simplicity, understandability and the ability for users to add their 
+own parts encompass a well-designed language. 
+
```

Comment:
just a grammar thing but the second sentence here is a bit difficult to understand. do you mean "We believe that a well-designed language should be simple, understandable and have the ability for users to add their own parts."?
that would be a much clearer way of saying that